components/user_settings/advanced/user_settings_advanced.jsx

```diff
@@ -140,7 +141,7 @@ export default class AdvancedSettingsDisplay extends React.Component {
 
         this.setState({isSaving: true});
 
-        await actions.savePreferences(userId, preferences);
+        actions.savePreferences(userId, preferences);
```

Comment:
I think this async/await for `handleSubmit` is needed so that the new preference will reflect at min view setting.  Could you please revert this unrelated change?

Reply:
oops, my inattention. I have reverted